cedar-policy-core/src/est/expr.rs

```diff
@@ -1243,36 +1295,34 @@ impl TryFrom<cst::Member> for Expr {
                                 }
                             }
                         }
-                        _ => {
-                            return Err(
-                                ParseError::ToAST("malformed method call".to_string()).into()
-                            )
-                        }
+                        _ => return Err(ParseError::MalformedMethodCall("".to_string()).into()),
```

Comment:
empty-string error message seems not ideal ... should we have a `String` in this error variant at all?

Reply:
The other uses have messages: `expected method name, found {}` and `this item does not have methods` which I think are useful. I can print out the item here, but I'm not sure how legible that will be. If it's a typo that's probably useful. If it's some complex AST node probably not.